scio-test/src/test/scala/com/spotify/scio/coders/JIterableWrapperSerializerTest.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.spotify.scio.coders
+
+import org.scalatest.FlatSpec
+
+class JIterableWrapperSerializerTest extends FlatSpec {
+
+  "JIterableWrapperSerializer" should "cope with the internal buffer overflow" in {
+    val ser = new JIterableWrapperSerializer[String](128)
+    val input = List.fill(100)("foo")
+    CoderTestUtils.testRoundTrip(ser, input)
+  }
+
+  it should "be able to serialize object larger than the internal buffer" in {
+    val ser = new JIterableWrapperSerializer[String](10)
+    val input = Seq("o" * 10)
```

Comment:
Nope - it does not have to be 10 bytes - just something (single object) that will overflow the 10 bytes buffer. I don't see how `>> 10` will make it more obvious, also the description of the test is pretty descriptive I think.